capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -336,6 +336,9 @@ public void onCreate(final Bundle savedInstanceState) {
         if (savedInstanceState != null) {
             restoreSavedState(savedInstanceState);
         }
+        // TODO Remove it after migration to GiniCapture.initialize(context)
+        UserAnalytics.INSTANCE.initialize(this.mFragment.getActivity().getApplicationContext());
```

Comment:
Added this for backward compability of using deprecated `GiniCapture.newInstance()`

Reply:
[SUGGESTION] I would suggest adding sth to make it more clear when this should be done. e.g. `        // TODO Remove it after migration to GiniCapture.initialize(context) - next major release (from version 3 to 4)

Reply:
Done